### DISCUSSION TITLE
现在逐字歌词和歌词辉光怎么没了

### DISCUSSION THREAD

**Discussion by KaitouKidgmh** [amll-dev/applemusic-like-lyrics#243] [General]
现在逐字歌词和歌词辉光怎么没了
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/124412473/e480cfc4-d2f6-404a-9632-6952baf91dc2)

**Comment by SteamFinder** [comment]
AMLL歌词来源主要有以下两种

- [AMLL歌词数据库](https://steamfinder.github.io/search-in-amlldb) 里面的歌词是特别优化过的 完美支持AMLL
- 网易云歌词数据库 由网易云提供 非TTML歌词 效果差 部分有逐字 大部分是逐行

你所播放的歌曲在[AMLL歌词数据库](https://steamfinder.github.io/search-in-amlldb) 中没有对应歌词，网易云提供的为逐行版本

**Comment by jiangyin14** [comment]
有大佬整理了q音的接口吗 可以拿来用的那种 感谢
